src/Microsoft.AspNet.Server.Kestrel/Http/Listener.cs

```diff
@@ -53,7 +53,7 @@ public Listener(IMemoryPool memory)
                     ListenSocket = new UvTcpHandle();
                     ListenSocket.Init(Thread.Loop, Thread.QueueCloseHandle);
                     ListenSocket.Bind(new IPEndPoint(IPAddress.Any, port));
-                    ListenSocket.Listen(10, _connectionCallback, this);
+                    ListenSocket.Listen(128, _connectionCallback, this);
```

Comment:
That's the socket listen backlog number, what would be better? an integer constant somewhere?